kolibri/deployment/default/settings/base.py

```diff
@@ -126,32 +125,41 @@
 # Database
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(KOLIBRI_HOME, 'db.sqlite3'),
-        'OPTIONS': {
-            'timeout': 100,
+if conf.OPTIONS['Database']["DATABASE_ENGINE"] == "sqlite":
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': os.path.join(KOLIBRI_HOME, conf.OPTIONS['Database']['DATABASE_NAME'] or 'db.sqlite3'),
```

Comment:
Since `DATABASE_NAME` is used for both sqlite and postgres, couldn't cleanly specify this default name in the options, so I'm leaving it here for now.